e2e/features/step_definitions/coreUX.stepdef.ts

```diff
@@ -5,9 +5,10 @@
 import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
 
 Given('I am on the strimzi-ui homepage', () => {
-  cy.visit('localhost:3000/index.html');
+  cy.visit('localhost:8080/index.html');
```

Comment:
Why the port change? When your run `npm run e2e` it should be starting up express on `3000`

Reply:
Was a mistake I will fix it